### PR TITLE
aprs.fi and aprsdirect.com

### PR DESCRIPTION
**Pull request by Luceoria**

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -1070,8 +1070,8 @@ INVERT
 aprs.fi
 
 INVERT
-.gm-style > :first-child
-.gm-style-iw-a
+img[src^="https://maps.googleapis.com/maps/"]
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1073]: Does this fixes any existing issues?

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -1080,6 +1080,14 @@ CSS
 
 ================================
 
+aprsdirect.com
+
+INVERT
+.leaflet-tile-pane
+img[src^="https://maps.googleapis.com/maps/"]
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1087]: I couldn't find such element, and would also probably be conflicting with the `leaflet-tile-pane`.

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Luceoria) [reply src/config/dynamic-theme-fixes.config:1073]: It does. Node icons are displayed inverted on the map, but not on the pop-up:
image
After the commit, both icon types are non-inverted.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:1073]: Both should be inverted. The non-inverted version doesn't look that good in dark-mode environment.

**Review by Luceoria — COMMENTED** [review]
Google Maps tiles are now inverted, so are icons.

**Comment by Luceoria** [comment]
Will submit another PR.

**Pull request closed without merging**